### PR TITLE
testing/topgit: new aport

### PR DESCRIPTION
**Pull request by jbenden**
https://github.com/mackyle/topgit
A different patch queue manager

Signed-off-by: Joseph Benden <joe@benden.us>

### CODE REVIEW

File: testing/topgit/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Contributor: Joseph Benden <joe@benden.us>
+# Maintainer: Joseph Benden <joe@benden.us>
+
+pkgname=topgit
+pkgver=0.19.12
+pkgrel=0
+pkgdesc="A different patch queue manager"
+arch="noarch"
+url="https://github.com/mackyle/topgit"
+license="GPL"
```
Comment (maxice8) [inline testing/topgit/APKBUILD]: Ambiguous and not SPDX compliant

File: testing/topgit/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+options="!check !strip"
+depends="git gawk sed"
+makedepends="gawk sed perl"
```
Comment (maxice8) [inline testing/topgit/APKBUILD]: gawk and sed are in depends= no need to be in makedepends

File: testing/topgit/APKBUILD (hunk 1 above)
Comment (jbenden) [reply testing/topgit/APKBUILD]: Fixed. Thank you for the review!

File: testing/topgit/APKBUILD (hunk 2 above)
Comment (jbenden) [reply testing/topgit/APKBUILD]: Ok, thanks!

**Pull request closed without merging**